gateway-spi/src/main/java/org/apache/knox/gateway/config/GatewayConfig.java

```diff
@@ -822,7 +822,7 @@ public interface GatewayConfig {
 
   Set<String> getPrivilegedUsers();
 
-  Set<String> getNonPrivilegedUsers();
+  Set<String> getUnlimitedUsers();
```

Comment:
We may rename this to `getSessionVerificationUnlimitiedUsers` and the previous one to `getSessionVerificationPrivilegedUsers`. What do you think?

Reply:
Okay, it gives a bit more information.